src/control/server/security.go

```diff
@@ -36,7 +36,7 @@ import (
 	"github.com/pkg/errors"
 )
 
-const moduleID int32 = C.DRPC_MODULE_SECURITY_AGENT
+const moduleID int32 = C.DRPC_MODULE_SEC
```

Comment:
This file goes away with #1229. Only modified here to keep everything compiling.